pkg/clients/cloudsql/cloudsql.go

```diff
@@ -19,109 +19,124 @@ package cloudsql
 import (
 	"strings"
 
+	"github.com/pkg/errors"
+
```

Comment:
Nit: whitespace.

Reply:
I’ll make sure to clean this up in #151 :)